specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2024-04-01-preview/managedHsm.json

```diff
@@ -0,0 +1,1737 @@
+        "innererror": {
+          "x-ms-client-name": "innerError",
+          "readOnly": true,
+          "description": "The inner error, contains a more specific error code.",
+          "$ref": "#/definitions/Error"
+        }
+      },
+      "description": "The server error.",
+      "type": "object"
+    },
+    "DeletedManagedHsm": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Azure Resource Manager resource ID for the deleted managed HSM Pool."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the managed HSM Pool."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type of the managed HSM Pool."
+        },
+        "properties": {
+          "$ref": "#/definitions/DeletedManagedHsmProperties",
+          "description": "Properties of the deleted managed HSM"
+        }
+      },
+      "type": "object"
+    },
+    "DeletedManagedHsmProperties": {
+      "properties": {
+        "mhsmId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource id of the original managed HSM."
+        },
+        "location": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location of the original managed HSM."
+        },
+        "deletionDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The deleted date."
+        },
+        "scheduledPurgeDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The scheduled purged date."
+        },
+        "purgeProtectionEnabled": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Purge protection status of the original managed HSM."
+        },
+        "tags": {
+          "readOnly": true,
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Tags of the original managed HSM."
+        }
+      },
+      "description": "Properties of the deleted managed HSM.",
+      "type": "object"
+    },
+    "DeletedManagedHsmListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeletedManagedHsm"
+          },
+          "description": "The list of deleted managed HSM Pools."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of deleted managed HSM Pools."
+        }
+      },
+      "description": "List of deleted managed HSM Pools",
+      "type": "object"
+    },
+    "MHSMGeoReplicatedRegion": {
+      "type": "object",
+      "properties": {
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Can there be a format applied here?

Reply:
Same as other comment, this is MHSM code that we dont own and I am just copying over their existing code.